examples/simple_model_example_4.yml

```diff
@@ -0,0 +1,28 @@
+sample:
+    model:
+        origin: softmatter example
+        stack: Si | LL | rLL | D2O
+        sub_stacks:
+            LL:
+                sequence:
+                - material: {sld: 1.88401254e-06}
+                  thickness: 9.0
+                  roughness: 3.0
+                - material: {sld: -3.73401535e-07}
+                  thickness: 1.4
+                  roughness: 3.0
+                represents:  refnx.reflect.LipidLeaflet
```

Comment:
This setup looks interesting. However, what happens if/when the parameterisation of a given function/class changes? Or more generally if there are changes between different versions of an analysis package?

Reply:
That's a good point, maybe such a description would need to include the software version, e.g. in the model "origin" attribute.

As alternative it is always possible we define our own "LipidLeaflet" model class that defines the slab-generation. It was just my understanding, that there is not really a consensus in the soft-matter community on how these are parameterized.
So this was a proposal how to deal with that in a generic way.